mkdocs.yml

```diff
@@ -11,52 +11,70 @@ markdown_extensions:
+  # Payment Processors: extensions/payment-processors.md # page-tree = NEED_NEW_PAGE
+  - Publishing Extensions: extensions/publish.md   # page-tree = DONE
+  - Extension Lifecycle: extend-stages.md          # page-tree = NEED_PAGE_MOVE to /extensions/lifecycle.md
+  - Troubleshooting: extensions/troubleshooting.md # page-tree = DONE
+# Framework Reference:
+  # Bootstrap: /framework/bootstrap.md          # page-tree = NEED_NEW_PAGE
+  # Cache: /framework/cache.md                  # page-tree = NEED_NEW_PAGE
+  # Components: /framework/components.md        # page-tree = NEED_NEW_PAGE
+  # Database: /framework/database.md            # page-tree = NEED_NEW_PAGE
+  # Resources: /framework/resources.md          # page-tree = NEED_NEW_PAGE
+  # Upgrade: /framework/upgrade.md              # page-tree = NEED_NEW_PAGE
+# Code Standards:
+  # PHP: /standards/php.md                      # page-tree = NEED_NEW_PAGE
+  # Javascript: /standards/js.md                # page-tree = NEED_NEW_PAGE
+  # API: /standards/api.md                      # page-tree = NEED_NEW_PAGE
+  # Git: /standards/git.md                      # page-tree = NEED_NEW_PAGE
+# Testing:
+  # Testing: /testing/testing.md                # page-tree = NEED_NEW_PAGE
+# Other Reference:
+  # CiviMail: /reference/civimail.md            # page-tree = NEED_NEW_PAGE
+  # CiviReport: /reference/civireport.md        # page-tree = NEED_NEW_PAGE
+  # Payment Processing: /reference/payment.md   # page-tree = NEED_NEW_PAGE
 - Hooks:
-   - How to Use Hooks: hook.md
-   - All Available Hooks: hooks-db.md
-- Writing Extensions:
-  - Basics: extensions/basics.md
-  - Using civix: extensions/civix.md
-  - Anatomy of Files: extensions/files.md
-  - Advanced Patterns: extensions/advanced.md
-  - Troubleshooting: extensions/troubleshooting.md
-  # - Using hooks: extensions/using-hooks.md
-  # - Customizing Screens: extensions/customize-screens.md
-  # - Creating Pages: Forms/extensions/create-page.md
-  # - Storing Configuration: extensions/config.md
-  # - Storing Data: extensions/storing-data.md
-  # - Setting Permissions: extensions/permissions.md
-  # - Custom API Functions: extensions/custom-api.md
-  # - Custom Reports: extensions/custom-reports.md
-  # - Custom Searches: extensions/custom-searches.md
-  # - Payment Processors: extensions/payment-processor.md
-  - Publish: extensions/publish.md
-- Best Practices:
-  - Documentation Style Guide: best-practices/documentation-style-guide.md
-- Core code:
-  - Hacking the core: core/hacking.md
-  - Architecture: core/architecture.md
-- Miscellaneous:
-   - Extension Lifecycle: extend-stages.md
-   - Markdown: markdownrules.md
-   # - hookref-old: hookref-old.md
-- Deprecated:
-   - Building CiviCRM from source: develop-deprecated.md
+  - Using hooks: hook.md                        # page-tree = NEED_PAGE_MOVE to /hooks/usage.md
+- MISC TO REORGANIZE OR DELETE:
+  - Extension advanced patterns: extensions/advanced.md
+  - Extensions files: extensions/files.md
```

Comment:
Agreed, but I'd like to take care of that later, and get this surface-level page reorganization in place first.